src/components/mol.box/box.styled.jsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import {Spacing} from '../constants'
+
+const HAlign = {
+    flexStart: 'flex-start',
+    flexEnd: 'flex-end',
+    center: 'center',
+    spaceBetween: 'space-between',
+    spaceAround: 'space-around',
+    spaceEvenly: 'space-evenly'
+}
+
+const VAlign = {
+    flexStart: 'flex-start',
+    flexEnd: 'flex-end',
+    center: 'center',
+    spaceBetween: 'space-between',
+    spaceAround: 'space-around',
+    spaceEvenly: 'space-evenly'
+}
+
+export const HBox = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: ${props => props.halign ? HAlign[props.halign] : 'center'};
+    align-items: ${props => props.valign ? VAlign[props.valign] : 'center'};
+    height: 100%;
+    width: ${props => props.grow ? '100%' : 'auto'};
+`;
+
+HBox.Item = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: ${props => props.halign ? HAlign[props.halign] : 'center'};
```

Comment:
hm n to vendo mta vantagem nesse HAlign. se for passado um valor q n está definido nele, ele vai retornar  undefined e n vai cair em nenhum fallback. Acho q pra servir do jeito esperado deveria ser:
```
${props => HAlign[props.halign] ?? 'center'};
```
assim, pro valor q n está dentro de HAlign, ele vai usar 'center' como fallback (me avisa se ficar com duvida nisso! posso detalhar mais)